book/forms.md

```diff
@@ -1194,7 +1189,7 @@ Each topic should have its own URL and its own list of messages. So,
 for example, `/cooking` should be a page of posts (about cooking) and
 comments submitted through the form on that page should only show up
 when you go to `/cooking`, not when you go to `/cars`. Make the home
-page, from `/`, list the available topics with a link to each topic's
```

Comment:
why?

Reply:
When reading I couldn’t understand what “from /“ means, and I guess it doesn’t seem too important to specify the URLs, the exercise is pretty open ended.

Reply:
I think it'd be better to revert this edit.

Reply:
Ok, new suggestion: change `from /` to `at /`